blazor/media-query/getting-started.md

```diff
@@ -0,0 +1,447 @@
+---
+layout: post
+title: Getting Started with Blazor Media Query Component | Syncfusion
+description: Checkout and learn about getting started with Blazor Media Query component in Blazor Server App and Blazor WebAssembly App.
+platform: Blazor
+control: Media Query
+documentation: ug
+---
+
+# Getting Started with Blazor Media Query Component
+
+This section briefly explains about how to include `Blazor Media Query` component in your Blazor Server App and Blazor WebAssembly App using Visual Studio. 
+
+## Prerequisites
+
+* [System requirements for Blazor components](https://blazor.syncfusion.com/documentation/system-requirements)
+
+## Create a new Blazor App in Visual Studio
+
+You can create **Blazor Server App** or **Blazor WebAssembly App** using Visual Studio in one of the following ways,
+
+* [Create a Project using Microsoft Templates](https://docs.microsoft.com/en-us/aspnet/core/blazor/tooling?pivots=windows)
+
+* [Create a Project using Syncfusion Blazor Extension](https://blazor.syncfusion.com/documentation/visual-studio-code-integration/create-project)
+
+## Install Syncfusion Blazor Layouts NuGet in the App
+
+Syncfusion Blazor components are available in [nuget.org](https://www.nuget.org/packages?q=syncfusion.blazor). In order to use Syncfusion Blazor components in the application, add reference to the corresponding NuGet. Refer to [NuGet packages topic](https://blazor.syncfusion.com/documentation/nuget-packages) for available NuGet packages list with component details and [Benefits of using individual NuGet packages](https://blazor.syncfusion.com/documentation/nuget-packages#benefits-of-using-individual-nuget-packages). 
+
+To add Blazor Media Query component in the app, open the NuGet package manager in Visual Studio (*Tools → NuGet Package Manager → Manage NuGet Packages for Solution*), search for [Syncfusion.Blazor.Layouts](https://www.nuget.org/packages/Syncfusion.Blazor.Layouts) and then install it.
```

Comment:
Syncfusion.Blazor.Core package, Layouts package not contained media query component.

Reply:
Hi @silambarasani ,

I modified the content as per discussion.

Regards,
@Backiaraj